template.yaml

```diff
@@ -355,14 +355,13 @@ Resources:
       LogGroupName: !Sub /aws/ecs/${AWS::StackName}-BAVFront-ECS
       RetentionInDays: 14
 
-  # CSLSECSAccessSubscriptionFilter:
-  #   Type: AWS::Logs::SubscriptionFilter
-  #   # Condition: IsNotDevelopment
-  #   Properties:
-  #     DestinationArn:
-  #       !FindInMap [PlatformConfiguration, !Ref Environment, CSLSEGRESS]
-  #     FilterPattern: ""
-  #     LogGroupName: !Ref ECSAccessLogsGroup
+  CSLSECSAccessSubscriptionFilter:
```

Comment:
There is a `CSLSAPIGWAccessSubscriptionFilter` commented out on line 652 can this be enabled now or no?